### PR TITLE
Remove deprecated codes related to ArrayTranslate

### PR DESCRIPTION
**Pull request by 0dvictor**
The old ArrayTranslate implementations are deprecated;
and hence they should be deleted.

Signed-off-by: Victor Ding <dvictor@ca.ibm.com>

### CODE REVIEW

File: runtime/tr.source/trj9/build/files/host/i386.mk [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
 JIT_PRODUCT_SOURCE_FILES+=\
     tr.source/trj9/x/i386/runtime/J9IA32ArrayCopy.asm \
-    tr.source/trj9/x/i386/runtime/J9IA32ArrayTranslate.asm \
     tr.source/trj9/x/i386/runtime/J9IA32CompressString.asm \
```
Comment (fjeremic) [inline runtime/tr.source/trj9/build/files/host/i386.mk]: Slightly related but it appears J9IA32CompressString.asm can be deprecated too.

Comment (0dvictor) [reply runtime/tr.source/trj9/build/files/host/i386.mk]: Thank-you for pointing it out, I'll look into clean it up later.

**Comment by andrewcraik** [comment]
Jenkins test sanity

**Comment by 0dvictor** [comment]
All Copyright Check failures are from `.gitignore` files. Refer to https://github.com/eclipse/openj9/issues/279 for details.

**Comment by andrewcraik** [comment]
Jenkins test sanity

**Comment by charliegracie** [comment]
@0dvictor can you add the copyright header to the `.gitignore'?  It should technically be there and it would resolve your issue.  You can copy the copyright header from any file that is using one with `#` as the comment prefix

**Comment by 0dvictor** [comment]
I'm going to add following copyright header to `.gitignore` as follow once Jenkins sanity test passed, to avoid redo sanity tests.
# Copyright (c) 2000, 2017 IBM Corp. and others
#
# This program and the accompanying materials are made available under
# the terms of the Eclipse Public License 2.0 which accompanies this
# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
# or the Apache License, Version 2.0 which accompanies this distribution and
# is available at https://www.apache.org/licenses/LICENSE-2.0.
#
# This Source Code may also be made available under the following
# Secondary Licenses when the conditions for such availability set
# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
# General Public License, version 2 with the GNU Classpath
# Exception [1] and GNU General Public License, version 2 with the
# OpenJDK Assembly Exception [2].
#
# [1] https://www.gnu.org/software/classpath/license.html
# [2] http://openjdk.java.net/legal/assembly-exception.html
#
# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0

**Comment by 0dvictor** [comment]
Copyright headers are updated via https://github.com/eclipse/openj9/pull/286, will rebase once #286 is merged.

**Comment by 0dvictor** [comment]
All sanity test have passed, and #286 is merged. Rebasing.

**Comment by andrewcraik** [comment]
Jenkins test sanity

**Comment by andrewcraik** [comment]
Jenkins test sanity

**Pull request merged**